codegen/templates/code/types/type_helpers.gotmpl

```diff
@@ -0,0 +1,16 @@
+// Definitions for the Kubernetes types
+package {{ .Version }}
+
+import (
+    metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+    {{- if $.CustomTypesImportPath }}
+    . "{{ $.CustomTypesImportPath }}"
+    {{- end}}
+)
+
+{{- range $resource := $.Resources }}
+
+// {{ $resource.Kind }}Slice represents a slice of *{{ $resource.Kind }}
+type {{ $resource.Kind }}Slice []*{{ $resource.Kind }}
```

Comment:
What do you think about putting a Sort method on this, or possibly on the sets?

Reply:
i don't follow; set is a map

and i'm happy to introduce a sort method here but i didn't need one. i think this PR is a bit large enough, but happy to approve it in a follow up